compiler/lookups.nim

```diff
@@ -634,7 +635,19 @@ proc qualifiedLookUp*(c: PContext, n: PNode, flags: set[TLookupFlag]): PSym =
         if m == c.module:
           result = strTableGet(c.topLevelScope.symbols, ident).skipAlias(n, c.config)
         else:
-          result = someSym(c.graph, m, ident).skipAlias(n, c.config)
+          var collides: bool
+          for i in 0..c.imports.high:
```

Comment:
```suggestion
          for i in 0 .. c.imports.high:
```

Reply:
O(n) for `module.foobar` where `n` is the number of imported modules? That's not good enough.

Reply:
It only runs on qualifications though and over locally imported modules. Shouldnt it be fine? Couldnt find any other hook. Tried with the module graph but it's empty at that point. 
Caching the name/ids pars or converting the seq of imported modules into a table is not an option, right? Any suggestions?